receiver/k8sobjectsreceiver/receiver.go

```diff
@@ -176,7 +201,9 @@ func (kr *k8sobjectsreceiver) startPull(ctx context.Context, config *K8sObjectsC
 		case <-ticker.C:
 			objects, err := resource.List(ctx, listOption)
 			if err != nil {
-				kr.setting.Logger.Error("error in pulling object", zap.String("resource", config.gvr.String()), zap.Error(err))
+				kr.setting.Logger.Error("error in pulling object",
```

Comment:
Shouldn't this be done using the `handleError` method?

Reply:
I would avoid handling all errors. I think `error_mode` was originally proposed for errors that occur while checking if desired k8s objects are available via the K8s API. These are errors that we can easily ignore or silent but we cannot easily generalise for all errors that might occur during the execution of the receiver. Maybe we can rename the setting or clearly mention this in the docs to avoid confusion. That said, I think we should only use  `handleError` for the respective k8s-api-objects-validation errors and leave everything else as is. Otherwise we introduce extra complexity to the receiver.

Reply:
I agree. When asked that, I was trying to understand on what situations we wanted to silent or not. But I totally agree with you